File: LeetCode_2125/Main.py
class Solution(object):
    def numberOfBeams(self, bank):
        """
        :type bank: List[str]
        :rtype: int
        """
        answer = 0
        last = 0
        for floor in bank:
            cur = 0
            for i in floor:
                cur += i == '1'
            answer += last * cur
            if cur:
                last = cur
        return answer

File: LeetCode_2125/Main.js
/**
 * @param {string[]} bank
 * @return {number}
 */
var numberOfBeams = function(bank) {
    let answer = 0;
    let last = 0;

    for (let s of bank) {
        let cur = 0;
        for (let ch of s) {
            cur += ch == '1';
        }

        if (cur) {
            answer += cur * last;
            last = cur;
        }
    }
    return answer;
};